docs/system-admin-guide/authentication/scim/README.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_navigation:
+  title: SCIM
+  priority: 800
+description: SCIM
+keywords: SCIM
+---
+# SCIM (Enterprise add-on)
+
+> [!IMPORTANT] 
+> SCIM is an Enterprise add-on. If you do not see the button you will have to activate the Enterprise edition first.
+
+To activate and configure SCIM in OpenProject, navigate to *Administration* -> *Authentication* and choose -> *SCIM*.
+
+## Configure new SCIM client.
+
+To add a new SCIM client, click the green **+ SCIM client** button.
+
+![Add SCIM client. Index page creation button.](add_scim_1.png) 
+
+
+You are able to configure the following options for your SCIM client:
+
+1. Enter the **Name** of your SCIM client.
+   ![Add SCIM client. Creation form. Name field.](add_scim_2.png) 
+2. Choose an **Authentication provider**. 
+   This is the service that users added by the SCIM provider will use to authenticate in OpenProject.
+   It should be available already in OpenProject. It can be [OIDC provider](../system-admin-guide/authentication/openid-providers/) or [SAML provider](../system-admin-guide/authentication/saml/).
+   ![Add SCIM client. Initial creation form. Authentication provider field.](add_scim_3.png) 
+3. Choose an **Authentication method**. 
+   This is how the SCIM client authenticates at OpenProject. Please ensure that OAuth tokens include the **scim_v2** scope.
+   There are three option:
+   a. **Static access token**
+      This is the most commomnly used authentication method for SCIM clients. In this case after clicking **Create** you get an access token that should be put to the SCIM client configuration on the other end.
+      ![Add SCIM client. Creation form. Static access token. Generate token. ](add_scim_4.png) 
+      There is generated access token. After closing the dialog with generated token you will not see it anymore.
+      ![Add SCIM client. Creation form. Static access token. Copy token.](add_scim_5.png) 
+      Token can be revoked. 
+      ![Add SCIM client. Creation form. Static access token. Revoke token.](add_scim_6.png) 
```

Comment:
An important hint about static access tokens is that they are valid for one year and must be replaced manually.
IMO we should indicate that in the docs as well.